### DISCUSSION TITLE
Hide tray icon

### DISCUSSION THREAD

**Discussion by Quijib0** [LizardByte/community#334] [Sunshine] [answered]
Hi,

I noticed someone was asking for system tray implementation across different platforms and the discussion was closed and adding it to MacOS appears on the roadmap.

However, I really don't want any icon at all.  I don't really find it useful and I can connect to the web UI just fine on my own.  I understand that others may like it.  In older releases (Win), running as a service got rid of the tray icon.  Now, with 0.21, it appears to be ever present.

Any chance on making this optional?

Thanks for your work on this.  It's a great alternative to Nvidia's.

**Comment by ReenigneArcher** [comment, marked as answer]
System tray will now be optional per changes in https://github.com/LizardByte/Sunshine/pull/4208
